From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ppatierno__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

itechedit committed:
edit pass: log-analytics-view-designer-parts

Lisaco88 committed:
edit pass: service-fabric-diagnostics-oms-setup

itechedit committed:
Update log-analytics-view-designer-parts.md

itechedit committed:
Update log-analytics-view-designer-parts.md

itechedit committed:
Update log-analytics-view-designer-parts.md

Ja-Dunn committed:
edit pass: GettingStarted

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into service-fabric-diagnostics-oms-setup

Lisaco88 committed:
edit pass: service-fabric-diagnostics-oms-setup

Ja-Dunn committed:
edit pass: GettingStarted

Ja-Dunn committed:
edit pass: GettingStarted

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into service-fabric-diagnostics-oms-setup

Lisaco88 committed:
edit pass: service-fabric-diagnostics-oms-setup

itechedit committed:
edit pass: log-analytics-view-designer-tiles

billmath committed:
sql delegation page

billmath committed:
adding toc

billmath committed:
adding object synch page

billmath committed:
adding toc entry

billmath committed:
adding gdpr page

jamesbak committed:
Update Azure Blob Storage capabilities in ADLS comparison article.

jamesbak committed:
Fixed URL reference

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into gdpr

billmath committed:
updating

billmath committed:
updating

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into objsync

billmath committed:
fixing acrolinix issues

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into gdpr

billmath committed:
fixing acrolinx issues

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into gdpr

billmath committed:
fixing formatting

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into gdpr

billmath committed:
updating

carolinacmoravia committed:
C11986: Extra tabulations breaking target format 

Hello, @garyericson, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

carolinacmoravia committed:
C11464: Extra ` in strucutrure

Hello, @navyasric,  

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

billmath committed:
updating

billmath committed:
adding health page

billmath committed:
adding cross-link

billmath committed:
acrolinx

billmath committed:
updating

billmath committed:
updating

billmath committed:
updating

billmath committed:
removing some screenshots

zhiweiwangmsft committed:
Update GDPR docs

Rephrase the data collection retention policy
For deletion action, remove the unnecessary screenshot and link to existing operation steps directory.

billmath committed:
updating

billmath committed:
updating

billmath committed:
Merge pull request #1 from zhiweiwangmsft/patch-5

Update GDPR docs

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into gdpr

billmath committed:
Merge branch 'gdpr' of https://github.com/billmath/azure-docs-pr into gdpr

billmath committed:
updating

MihaelaVorvoreanu committed:
Update Home.md

tamram committed:
updating account creation include for blob qs

gsacavdm committed:
Add ARM_ENVIRONMENT

KumudD committed:
added connection monitor file

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into connectionmonitor

KumudD committed:
changed file name

Commit:
Merge pull request #2 from MicrosoftDocs/master

Synch with head fork

Lisaco88 committed:
Made two requested changes

KumudD committed:
toc update

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into connectionmonitor

spelluru committed:
Cosistent naming for the linekd service

KumudD committed:
PR review incorp

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into connectionmonitor

ShawnJackson committed:
Update service-fabric-diagnostics-oms-setup.md

ShawnJackson committed:
Merge pull request #33025 from Lisaco88/service-fabric-diagnostics-oms-setup

edit pass: service-fabric-diagnostics-oms-setup

WilliamAntonRohm committed:
Merge pull request #74 from MicrosoftDocs/master

howdy

WilliamAntonRohm committed:
added entry for Cosmos ARM templates

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into gdpr

billmath committed:
fixing acrolinx'

billmath committed:
acrolinxs

Lisaco88 committed:
Merge pull request #33542 from KumudD/connectionmonitor

Connection monitor

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into sql

billmath committed:
fixing per feedback

v-shils committed:
Merge pull request #33298 from billmath/objsync

Objsync

itechedit committed:
Update log-analytics-view-designer-parts.md

gunjanj committed:
Updated provisioning failed section

PS cmdliet snipped fix based on Andre's comments

PRMerger18 committed:
Merge pull request #33553 from spelluru/sproc-0216

Consistent naming for the linekd service

Commit:
Merge pull request #33464 from carolinacmoravia/patch-617

C11464: Extra ` in strucutrure

Commit:
Merge pull request #33463 from carolinacmoravia/patch-616

C11986: Extra tabulations breaking target format

itechedit committed:
Update log-analytics-view-designer-parts.md

itechedit committed:
Update log-analytics-view-designer-parts.md

PRMerger15 committed:
Merge pull request #33557 from gunjanj/patch-8

Updated provisioning failed section

v-shils committed:
Merge pull request #33554 from WilliamAntonRohm/W-cosmos-arm

adding Cosmos ARM template entry

solankisamir committed:
Update ValidateJWT info

Lisaco88 committed:
Added a word space

ShawnJackson committed:
Merge pull request #33018 from itechedit/log-analytics-view-designer-parts

edit pass: log-analytics-view-designer-parts

solankisamir committed:
few typo fixes

v-shils committed:
Merge pull request #33354 from jamesbak/master

Update Azure Blob Storage capabilities in ADLS comparison article.

billmath committed:
updating

billmath committed:
updating

ShawnJackson committed:
Merge pull request #33114 from Ja-Dunn/GettingStarted

edit pass: GettingStarted

PRMerger18 committed:
Merge pull request #33560 from solankisamir/master

Update ValidateJWT info

ShawnJackson committed:
Update log-analytics-view-designer-tiles.md

CarlRabeler committed:
new portal

Lisaco88 committed:
Merge pull request #33290 from billmath/sql

Sql

Lisaco88 committed:
Merge pull request #33302 from billmath/gdpr

adding gdpr page

ShawnJackson committed:
Merge pull request #33135 from itechedit/log-analytics-view-designer-tiles

edit pass: log-analytics-view-designer-tiles

CarlRabeler committed:
update image

tamram committed:
fix merge conflict

Commit:
Merge pull request #33563 from billmath/aadc

updating

tamram committed:
fix merge conflict

dkamstra committed:
Updated screenshots to reflect latest functionality

cwatson-cat committed:
Adding back in New Relic for AD integration.

MikeDodaro committed:
Web Java SDK quickstart

KumudD committed:
renamed file

rjagiewich committed:
Merge pull request #33565 from CarlRabeler/new-portal

new portal

Commit:
Merge pull request #33525 from MihaelaVorvoreanu/patch-1

Update Home.md

MikeDodaro committed:
Fix description text and spelling error.

zhiweiwangmsft committed:
Update active-directory-aadconnect-health.md

Lisaco88 committed:
Merge pull request #33569 from dkamstra/master

Updated screenshots and content to reflect latest functionality

rjagiewich committed:
Merge pull request #33528 from tamram/tamram-0215

updating account creation include for blob qs

Lisaco88 committed:
Merge pull request #33571 from cwatson-cat/2-16-18-RestoreNewRelicAD

Adding back in New Relic for AD integration.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #33572 from KumudD/filenameupdate

renamed file

v-alje committed:
Merge pull request #33577 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger19 committed:
Merge pull request #33575 from zhiweiwangmsft/patch-6

Update active-directory-aadconnect-health.md

rjagiewich committed:
Merge pull request #33573 from MikeDodaro/web-sdk-java-quickstart

Web Java SDK quickstart

PRMerger19 committed:
Merge pull request #33530 from gsacavdm/patch-4

Add ARM_ENVIRONMENT
